resource_obmcs_core_instance_test.go

```diff
@@ -109,6 +117,91 @@ func (s *ResourceCoreInstanceTestSuite) TestAccResourceCoreInstance_basic() {
 				}`,
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(s.ResourceName, "display_name", "-tf-instance"),
+					func(ts *terraform.State) (err error) {
+						newId, err := fromInstanceState(ts, s.ResourceName, "id")
+						if newId != instanceId {
+							return fmt.Errorf("Expected same instance ocid, got the different.")
+						}
+						return err
+					},
+				),
+			},
+			// verify force refresh by creating an instance with create_vnic_details.
```

Comment:
Do you mean "force refresh" or "force new" or "recreated"?

Reply:
force new, updated